include/glow/IR/IR.h

```diff
@@ -285,47 +327,29 @@ class IRFunction final {
   /// \returns the list of instructions.
   const InstListTy &getInstrs() const { return instrs_; }
 
+  /// getSublistAccess() - returns pointer to member of instruction list
```

Comment:
Why do you put the name of the function in the documentation? 

Also, Why is this called getSublistAccess ? Why not getInstructions() or something? What part is 'sub' ?

Reply:
Copy&paste from LLVM code. Let's give it a more reasonable name.

Reply:
Fixed.